javascript/src/webhook.ts

```diff
@@ -75,72 +36,20 @@ export class Webhook {
       headers[key.toLowerCase()] = (headers_ as Record<string, string>)[key];
     }
 
-    let msgId = headers["svix-id"];
-    let msgSignature = headers["svix-signature"];
-    let msgTimestamp = headers["svix-timestamp"];
-
-    if (!msgSignature || !msgId || !msgTimestamp) {
-      msgId = headers["webhook-id"];
-      msgSignature = headers["webhook-signature"];
-      msgTimestamp = headers["webhook-timestamp"];
-
-      if (!msgSignature || !msgId || !msgTimestamp) {
-        throw new WebhookVerificationError("Missing required headers");
+    for (const h of ["id", "signature", "timestamp"]) {
```

Comment:
Btw, you can probably change both Python and here to iterate on tuples like `("webhook-id", "svix-id")` and then do this. Will be slightly less repetitive and also clear.

It's just that the dynamically generating strings is bad.